includes/Modules/Idea_Hub.php

```diff
@@ -650,12 +663,15 @@ protected function setup_assets() {
 				)
 			),
 			new Script(
-				'googlesitekit-idea-hub-post-list-notice',
+				'googlesitekit-idea-hub-post-list',
 				array(
-					'src'           => $base_url . 'js/googlesitekit-idea-hub-post-list-notice.js',
+					'src'           => $base_url . 'js/googlesitekit-idea-hub-post-list.js',
 					'load_contexts' => array( Asset::CONTEXT_ADMIN_POSTS ),
 					'dependencies'  => array(
+						'googlesitekit-i18n',
 						'googlesitekit-datastore-user',
+						'googlesitekit-datastore-ui',
+						'googlesitekit-modules-idea-hub',
```

Comment:
Is the module needed here? I don't see it used in the tour.

Reply:
We check whether the idea hub module is active and connected.

Reply:
Yes, good catch. Updated.